python/ml4ir/base/config/parse_args.py

```diff
@@ -148,7 +148,21 @@ def define_args(self):
             "--run_id",
             type=str,
             default="",
-            help="Run ID for the current training. Autogenerated if not specified.",
+            help="Unique string identifier for the current training run. Used to identify logs and models directories. Autogenerated if not specified.",
+        )
+
+        self.add_argument(
+            "--run_group",
+            type=str,
+            default="general",
+            help="Unique string identifier to group multiple model training runs. Allows for defining a meta grouping to filter different model training runs for best model selection as a post step.",
+        )
```

Comment:
[very minor] Would be great to have an example here of what syntax is expected in order to use this arg.

Reply:
It's just a string actually. Not sure I understand what you mean by example.

Reply:
@lastmansleeping Oh I understand. It sounded like it did the grouping somehow. Ok, then it's ok as it is.